package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spidersuite",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
I bumped now, as we've added a feature that is not backwardly available. I tend to do so on small, less formal projects.